Dockerfile

```diff
@@ -106,8 +110,11 @@ RUN --mount=type=secret,id=github_private_cloud_token \
 # * api-runtime
 FROM python-slim as api-runtime
 
+# Uninstall pip to reduce CVE-2018-20225 noise
+# and install binary dependencies required for ARM64
 ARG TARGETARCH
-RUN if [ "${TARGETARCH}" != "amd64" ]; then \
+RUN pip uninstall -y pip && \
+  if [ "${TARGETARCH}" != "amd64" ]; then \
   apt-get update && apt-get install -y libpq-dev && rm -rf /var/lib/apt/lists/*; fi;
```

Comment:
I wonder if we still need to install these for arm?

Reply:
We might not https://github.com/psycopg/psycopg2/pull/1545

Reply:
E2E passes on ARM without it 👍